src/objects/aff/zcl_abapgit_object_common_aff.clas.abap

```diff
@@ -430,6 +432,16 @@ CLASS ZCL_ABAPGIT_OBJECT_COMMON_AFF IMPLEMENTATION.
           RECEIVING
             result = lo_object_json_file.
 
+        CALL METHOD lo_object_json_file->('IF_AFF_FILE~IS_DELETION')
+          RECEIVING
+            result = lv_is_deletion.
+
+        " avoid to serialize empty content (object was never activated, exists inactive only).
+        IF lv_is_deletion = abap_true.
```

Comment:
Minor improvement suggestion: Shall we move this into dedicated method (like `is_file_with_content` or similar) so that the intention is easier to understand?

```suggestion
        IF is_file_with_content( lo_object_json_file ) = abap_false.
```

Reply:
Done.